pkg/server/server.go

```diff
@@ -1133,6 +1225,20 @@ If problems persist, please see ` + base.DocsURL("cluster-setup-troubleshooting.
 		return err
 	}
 	log.Event(ctx, "started node")
+	s.startMonitoringClockJumps(ctx)
+	s.startPersistingFutureWallTime(
```

Comment:
I see a risk here. If reaching here takes more than `persistFutureWallTimeDelta`, then the server would crash. This is because `EnsureInitialWallTimeMonotonicity` persists a future wall time, and the cron to persist subsequent future wall times starts here.

We cannot start the cron earlier because `EnsureInitialWallTimeMonotonicity` only persists in bootstrapped stores, whereas the cron has to run on all stores.